examples/pytorch_lightning/msn.py

```diff
@@ -87,20 +85,16 @@ def configure_optimizers(self):
 
 model = MSN()
 
-# we ignore object detection annotations by setting target_transform to return 0
-pascal_voc = torchvision.datasets.VOCDetection(
-    "datasets/pascal_voc", download=True, target_transform=lambda t: 0
+transform = MSNTransform()
+dataset = pascal_voc = torchvision.datasets.VOCDetection(
+    "datasets/pascal_voc", download=True, transform=transform
 )
```

Comment:
```suggestion
dataset = torchvision.datasets.VOCDetection(
    "datasets/pascal_voc", download=True, transform=transform
)
```

Is `target_transform` not needed anymore?